tests/ert/unit_tests/gui/ertwidgets/models/test_ertsummary.py

```diff
@@ -91,11 +95,63 @@ def test_getForwardModels(mock_ert):
 
 def test_getParameters(mock_ert):
     expected_list = ["DEFAULT (3)", "field (10, 5, 3)", "surface (10, 7)"]
-    parameter_list, parameter_count = ErtSummary(mock_ert).getParameters()
+    parameter_list, parameter_count = ErtSummary(mock_ert).get_parameters()
     assert parameter_list == expected_list
     assert parameter_count == 223
 
 
+def test_that_design_matrix_parameters_are_included_in_the_parameter_count(mock_ert):
+    # Add design matrix parameters
+    dm_param1 = GenKwConfig(
+        name="dm_param_a",
+        distribution={"name": "uniform", "min": 0, "max": 1},
+        group="DESIGN_MATRIX",
+    )
+    dm_param2 = GenKwConfig(
+        name="dm_param_b",
+        distribution={"name": "uniform", "min": 0, "max": 1},
+        group="DESIGN_MATRIX",
+    )
+    dm_param3 = GenKwConfig(
+        name="dm_param_c",
+        distribution={"name": "uniform", "min": 0, "max": 1},
+        group="DESIGN_MATRIX",
+    )
+
+    # Modify the mock to return parameters including design matrix
+    mock_ert.parameter_configurations_with_design_matrix = [
+        *mock_ert.ensemble_config.parameter_configs.values(),
+        dm_param1,
+        dm_param2,
+        dm_param3,
+    ]
+
+    parameter_list, parameter_count = ErtSummary(mock_ert).get_parameters()
+
+    # Check that design matrix parameters are counted
+    assert "DESIGN_MATRIX (3)" in parameter_list
+    # Original count (223) + 3 design matrix parameters
+    assert parameter_count == 226
+
+
+@pytest.mark.usefixtures("use_tmpdir")
+def test_design_matrix_parameters_counted_when_loaded_from_real_config(
```

Comment:
A small thing, but this test name reads a bit better with a `that` inserted. Something like:

```diff
- def test_design_matrix_parameters_counted_when_loaded_from_real_config(
+ def test_that_design_matrix_parameters_are_counted_when_loaded_from_a_real_config(
```